internal/translation/deployment/deployment.go

```diff
@@ -64,8 +65,12 @@ func (ds *ProductionAtlasDeployments) ListDeploymentConnections(ctx context.Cont
 	}
 	clusterConns := clustersToConnections(clusters.GetResults())
 
-	serverless, _, err := ds.serverlessAPI.ListServerlessInstances(ctx, projectID).Execute()
-	if err != nil {
+	if ds.isGov {
+		return clusterConns, nil
+	}
```

Comment:
is this case governed by unit tests?

Reply:
I don't think we have unit test for the ProductionAtlasDeployments

Reply:
in that case, please add a unit test here to have coverage.

Reply:
Unit-test added